fragments/k8s/cdk/bundle.yaml

```diff
@@ -27,6 +27,8 @@ services:
     annotations:
       "gui-x": "450"
       "gui-y": "550"
+    to:
+      - lxd:kubernetes-master
```

Comment:
@CalvinHartwell Same question here - is this what's being done in customer engagements? How well tested is this? I'm okay with it in theory, just want to know what the confidence level is here.

Reply:
Yes, easyrsa does not belong on bare metal or a VM. We had this in the original CDK design, somewhere along the way it was removed

Reply:
@marcoceppi How does this work when people are adding/removing kubernetes-master units to scale their operations? Do they need to be careful to not remove the master with easyrsa on it?

Reply:
Indeed, if the machine with easyrsa crashes we will get into trouble. Perhaps this is an argument for keeping easyrsa separate.